cladecombiner/nomenclature.py

```diff
@@ -18,15 +18,17 @@ class Nomenclature(ABC):
     """
     Abstract class for most general casting of Nomenclature
 
-    Nomenclature concerns rules for naming taxa, and what names may imply about those taxa.
+    Nomenclature concerns rules for naming taxa, and what names may imply about
+    those taxa.
     """
 
     @abstractmethod
     def is_ambiguous(self, name: str) -> bool:
         """
         Does this name indicate an ambiguous taxon?
 
-        Ambiguity means a taxon known only to a higher level than to which resolution is possible.
+        Ambiguity means a taxon specified only to a higher level than to which
```

Comment:
Sorry, I'm still stuck on this. What do "specified" and "resolution" mean? Can you give a concrete example so I can understand what this means?

Reply:
Apologies, thought you were making a different sort of clarity suggestion. 

This is about things that get `*` in the name, for example, the row in the [Pango SC2 alias key](https://github.com/cov-lineages/pango-designation/blob/master/pango_designation/alias_key.json) that reads `"XD": ["B.1.617.2*","BA.1*"]`. This means "XD is a recombinant, of some child lineage of B.1.617.2 and some child lineage of BA.1." (We're ignoring the fact that the order here tells us what order the parentage is across the genome.)

Reply:
This is helpful. Is the language here that BA.1* is "specified" only to BA.1, which is a "higher" level than that to which BA.1* is "resolved"?

My current understanding is that `XD.get_parents()` would yield a list of length two, and that `parent1.is_ambiguous()` would give `TRUE`. Did I get that right?

And then this matters because, if I'm thinking about getting histories, I would need to account for the fact that `BA.1*` is somewhere on the tree/graph between `BA.1` and `XD`?

We should probably talk about this synchronously.